api/src/main/java/app/Multiplicacao.java

```diff
@@ -0,0 +1,23 @@
+package app;
+
+import io.jooby.annotations.*;
+
+import io.jooby.exception.BadRequestException;
+ 
+@Path("/multiplicacao/{op}/{op2}")
+public class Multiplicacao {
+  
+   @GET
+   public double calcula(@PathParam("op") String opStr, @PathParam("op2") String opStr2) {
+  
```

Comment:
Imprimir uma mensagem informando que quando um dos valores for 0, o produto também será 0.